core/src/main/scala/com/databricks/labs/remorph/generators/sql/LogicalPlanGenerator.scala

```diff
@@ -8,7 +8,18 @@ class LogicalPlanGenerator(val expr: ExpressionGenerator, val explicitDistinct:
     extends Generator[ir.LogicalPlan, String] {
 
   override def generate(ctx: GeneratorContext, tree: ir.LogicalPlan): String = tree match {
-    case b: ir.Batch => b.children.map(generate(ctx, _)).mkString("", ";\n", ";")
+    case b: ir.Batch =>
+      b.children
```

Comment:
refactor this branch into `batch(ctx, b)` private method

Reply:
done